Makefile

```diff
@@ -245,3 +245,6 @@ doc:
 .PHONY: fuzz
 fuzz:
 	@ ./scripts/gear.sh test fuzz
+
+build-wat-examples:
```

Comment:
please, move to "build" section to keep things together

Reply:
Done